src/interfaces/routes.ts

```diff
@@ -485,6 +485,29 @@ export interface AdjustLeverageResponse {
   maxNotionalValue: string;
 }
 
+export interface SubAccountRequest {
+  accountAddress: string;
+  accountsToRemove: Array<string>;
+}
+
+export interface SignedSubAccountRequest extends SubAccountRequest {
+  signedTransaction: string;
+}
+
+export interface SubAccountResponse {
+  userAddress: string;
+  txIndex?: number;
+  logIndex?: number;
+  blockNumber?: number;
+  id: number;
+  symbol: string;
```

Comment:
I don't think we need symbol here as on Sui, subaccounts are whitelisted for all markets if I remember correctly